TaskSharper.DataAccessLayer.Google/Calendar/Service/GoogleCalendarService.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Services;
+using TaskSharper.Domain.Calendar;
+
+namespace TaskSharper.DataAccessLayer.Google.Calendar.Service
+{
+    public class GoogleCalendarService : BaseService, ICalendarService
+    {
+        private readonly CalendarService _service;
+        public GoogleCalendarService(CalendarService service = null)
+        {
+            Authenticate();
+            if (service == null)
+            {
+                _service = new CalendarService(new BaseClientService.Initializer
+                {
+                    HttpClientInitializer = UserCredential,
+                    ApplicationName = "TaskSharper"
```

Comment:
Maybe consider to seperate the value "TaskSharper" into a _const string_ defined in the class, OR making a seperate .cs file (Constants.cs), defining it there and making the value static?

Reply:
Made Constants.cs and inserted the value there.